automat/_methodical.py

```diff
@@ -75,14 +75,27 @@ def _transitionerFromInstance(oself, symbol, automaton):
         setattr(oself, symbol, transitioner)
     return transitioner
 
+def assert_no_code(inst, attribute, f):
```

Comment:
Should be `assertNoCode`, but you can submit that fix in a follow up - I'll allow it, since this is private implementation details.